src/Configuration.h

```diff
@@ -82,6 +82,8 @@ class Configuration
   byte EE_NUM_EVS = 0;
   byte EE_PRODUCED_EVENTS = 0;
   byte EE_BYTES_PER_EVENT; // Value calculated in begin()
+  unsigned int EE_FREE_BASE; // Value calculated in begin()
+  unsigned int EE_USER_BYTES = 0; // Specified by user in setup for ESP processors.
```

Comment:
Oh, this one needs documentation so that users don't miss this.

Reply:
Looked at PersistentStorage.md and it needs a lot of work. It doesn't take into account that we now provide calculated defaults.
Yes, a How it Works would be nice.